docs/management/sentinel.md

```diff
@@ -847,7 +847,7 @@ configured with `requirepass`, the Sentinel configuration must include the
 Configuring Sentinel instances with authentication
 ---
 
-Sentinel instances themselves can be secured by requiring clients to authenticate via the `AUTH` command. Starting with Redis 6.2, the [Access Control List (ACL)](/topics/acl) is available, whereas previous versions (starting with Redis 5.0.1) support password-only authentication. 
+Sentinel instances themselves can be secured by requiring clients to authenticate via the `AUTH` command. PLEASE-REVIEW{Starting with Redis 6.2, the [Access Control List (ACL)](/topics/acl) is available, whereas previous versions (starting with Redis 5.0.1) support password-only authentication. }
```

Comment:
If it's very old (like redis 2 or 3) it is always fine to just delete it. If it's newer and can be useful to anyone, we can keep it with the terminology "Redis OSS".

If you're not sure, it is always possible to describe old versions as Redis OSS. It is the terminology accepted by Redis Trademark Policy. There is no need to ask every time.

```suggestion
Sentinel instances themselves can be secured by requiring clients to authenticate via the `AUTH` command. Starting with Redis OSS 6.2, the [Access Control List (ACL)](/topics/acl) is available, whereas previous versions (starting with Redis OSS 5.0.1) support password-only authentication.
```

Reply:
done, thanks for the clarification